### PR TITLE
Deployment

### PR DESCRIPTION
**Pull request by SergSkol**
-Implement all project requirements
-Test app by all requirements
-Make optimization
-Improve UI
-Add video presentation in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,10 +73,16 @@ Example command:
...
+  ruby main.rb
 ```
 
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📽️ Video Presentation <a name="video"></a>
+
+- [Video Presentation 📽️](https://drive.google.com/file/d/1Jl1U_LszW6nj9ob0WsgXn-VBVgYjBNud/view?usp=share_link)
+
```
Comment (AnselemOdims) [inline README.md]: - [ ] Good job on the presentation video but please kindly ensure that it does not exceed the time limit of 5minutes

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,323 @@
+require_relative './class/music_album'
+require_relative './class/genre'
+require_relative './class/book'
+require_relative './class/label'
+require_relative './class/game'
+require_relative './class/author'
```
Comment (AnselemOdims) [inline app.rb]: - [ ] At the method, importing these modules returns an error that is caused by wrong naming. The folder name `Class` and not `class`. Please kindly help fix this.

File: Database/schema.sql [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  id serial PRIMARY KEY,
+  publish_date DATE NOT NULL,
+  archived BOOLEAN NOT NULL,
+  genre_id INT,
+  author_id INT,
+  label_id INT,
+  PRIMARY KEY (id)
+  FOREIGN KEY (genre_id) REFERENCES genres(id),
+  FOREIGN KEY (author_id) REFERENCES authors(id),
+  FOREIGN KEY (label_id) REFERENCES labels(id),
```
Comment (AnselemOdims) [inline Database/schema.sql:48]: - [ ] Good job working on the database but please make sure all foreign keys columns have corresponding indexes.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @team, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (SergSkol) [reply README.md]: Thanks! It's done!

File: app.rb (hunk 2 above)
Comment (SergSkol) [reply app.rb]: Thanks! It's done!

File: Database/schema.sql (hunk 3 above)
Comment (SergSkol) [reply Database/schema.sql:48]: Thanks! It's done!

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi TEAM
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Well structured files :white_check_mark:
 * No Linter errors :white_check_mark:
 * Good Job :+1: 
 
 Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :


 
 ## Optional suggestions

- [x] N/A keep doing great job :clap: 

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**